Apps/WebClient/src/Server/Services/UserEmailService.cs

```diff
@@ -143,7 +143,7 @@ public bool UpdateUserEmail(string hdid, string email, Uri hostUri, string beare
             if (!string.IsNullOrEmpty(email))
             {
                 this.logger.LogInformation($"Sending new email invite for user ${hdid}");
-                this.emailQueueService.QueueNewInviteEmail(hdid, email, hostUri);
+                this.emailQueueService.QueueNewInviteEmail(hdid, email, hostUri, emailInvite != null ? emailInvite.InviteKey : null);
```

Comment:
As specified in the story it can only reuse the invite key if the new email address is the same as the previous one

Reply:
@tiagograf-AOT thanks, I though this code was called when user updates to the new email address. Forgot that it got called also for ReSend scenario.

Reply:
code updated